code/style.css

```diff
@@ -1,7 +1,247 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+    margin: 0 auto;
+    height: 100%;
+    background: #f9f5e9;
+    font-family: 'Fira Sans', sans-serif;
+    font-Size: 15px;
+}
+
+
+/* --- header-section --- */
+
+.hero-image {
+    position: relative;
+    height: 350px;
+    overflow: hidden;
+    background: url(./img/relax1.jpeg) no-repeat center;
+    background-size: cover;
+}
```

Comment:
Perfect styling for a hero image, it looks great on all devices too. Also a huge plus for using a separate image folder to structure your project 🏅